shared/components/events/whats-new.tsx

```diff
@@ -0,0 +1,40 @@
+import Fade from 'react-reveal/Fade'
+
+import Card from '@shared/components/events/card'
+
+const WhatsNew = ({ cardContent }) => (
```

Comment:
Add type here as well, you can create a common file containing all types and import wherever required